### PR TITLE
Accept system_type via interactive mode

### PR DESCRIPTION
**Pull request by Prajyot-Parab**
Fixes: #149
Signed-off-by: Prajyot Parab <Prajyot.Parab1@ibm.com>

### CODE REVIEW

**Comment by Prajyot-Parab** [comment]
CC - @bpradipt

**Review by bpradipt — APPROVED** [review]
/lgtm

File: docs/variables.md [hunk 1]
```diff
@@ -88,6 +88,11 @@ Targeting service crn:v1:bluemix:public:power-iaas:tor01:a/65b64c1f1c2XXXXX:4a1f
 1) ocp-net
 #? 1
 - You have answered: ocp-net
+[question] > Select the system type to use for cluster nodes:
+1) e980
+2) s922
+#? 2
```
Comment (yussufsh) [inline docs/variables.md:94]: E880 is also part of some powervs regions.. how are we gonna handle it if user selects 'e980' there?
@bpradipt

Comment (yussufsh) [reply docs/variables.md:94]: https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-about-virtual-server

Comment (Prajyot-Parab) [reply docs/variables.md:94]: @yussufsh had discussed regarding the same with @bpradipt 
Dallas and Washington has e880 - did not filter it out as if user selects the same in those region we expect it to work in theoretical terms (if this is something that needs to be avoided let me know, I can filter to exclude it)

Comment (yussufsh) [reply docs/variables.md:94]: You can validate the theory or fix it for the given regions.

Comment (Prajyot-Parab) [reply docs/variables.md:94]: Had asked Aishwarya to create cluster with e880 -- seems to be working fine no issues there. (will ask validation team to do some testing)

**Pull request merged**